proto_generator/protogenerator.go

```diff
@@ -100,7 +101,7 @@ func main() {
 	}
 
 	// Perform the code generation.
-	cg := ygen.NewYANGCodeGenerator(&ygen.GeneratorConfig{
+	cg := protogen.NewProtoCodeGenerator(&ygen.GeneratorConfig{
```

Comment:
Similar comments to the Go refactor, `protogen.NewGenerator` or even `protogen.New` might be more succinct here, since I expect that there's likely not much more than the generator in the package here.